### ISSUE TITLE
[bug]  import failure `i is not a function`

### ISSUE THREAD

**Issue by LiangChengming** [marp-team/marp-core#322]
```javascript
import { Marp } from "@marp-team/marp-core";
```

caused error:

<img width="1411" alt="image" src="https://user-images.githubusercontent.com/45283486/206611890-53ff82f0-4a33-4db0-b2e8-e18fa4a8a785.png">

**Comment by yhatt** [comment]
`i` means [`emoji-regex` module](https://github.com/mathiasbynens/emoji-regex). Is this code throwing some errors in your code context?

```javascript
import emojiRegex from 'emoji-regex'
```

**Comment by LiangChengming** [comment]
> `i` means [`emoji-regex` module](https://github.com/mathiasbynens/emoji-regex). Is this code throwing some errors in your code context?
> 
> ```js
> import emojiRegex from 'emoji-regex'
> ```

No error for this import

**Comment by yhatt** [comment]
OK, could you share a minimal reproducible example project? A code you've described is working on Node.js module without any errors.

If you are using a bundler, it may have met some kind of troubles about CommonJS interop from ESM.

**Comment by yhatt** [comment]
We have met exactly same situation in [marp-team/marp-vscode](https://github.com/marp-team/marp-vscode). Here is a workaround:
https://github.com/marp-team/marp-vscode/blob/23e905b970c81629353be31b564d50aadb2a8bc2/webpack.base.config.js#L7-L13

#323 may resolve bundler's confusion about default export in external emoji modules.

**Comment by yhatt** [comment]
Confirmed it was fixed in [v3.4.1](https://github.com/marp-team/marp-core/releases/tag/v3.4.1).
